src/proposals/20250529/SparkEthereum_20250529.sol

```diff
@@ -0,0 +1,239 @@
+        PAYLOAD_UNICHAIN = 0xbF5a7CfaF47fd1Ad75c9C613b1d4C196eE1b4EeF;
+    }
+
+    function _postExecute() internal override {
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveInterestRateStrategyAddress(Ethereum.DAI,  DAI_USDS_IRM);
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveInterestRateStrategyAddress(Ethereum.USDS, DAI_USDS_IRM);
+
+        RateLimitHelpers.setRateLimitData(
+            MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_USDS_MINT(),
+            Ethereum.ALM_RATE_LIMITS,
+            RateLimitData({
+                maxAmount : 500_000_000e18,
+                slope     : 500_000_000e18 / uint256(1 days)
+            }),
+            "usdsMintLimit",
+            18
+        );
+        RateLimitHelpers.setRateLimitData(
+            MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_USDS_TO_USDC(),
+            Ethereum.ALM_RATE_LIMITS,
+            RateLimitData({
+                maxAmount : 500_000_000e6,
+                slope     : 300_000_000e6 / uint256(1 days)
+            }),
+            "swapUSDSToUSDCLimit",
+            6
+        );
+        RateLimitHelpers.setRateLimitData(
+            MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_USDE_MINT(),
+            Ethereum.ALM_RATE_LIMITS,
+            RateLimitData({
+                maxAmount : 250_000_000e6,
+                slope     : 100_000_000e6 / uint256(1 days)
+            }),
+            "ethenaMintLimit",
+            6
+        );
+        RateLimitHelpers.setRateLimitData(
+            MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_USDE_BURN(),
+            Ethereum.ALM_RATE_LIMITS,
+            RateLimitData({
+                maxAmount : 500_000_000e18,
+                slope     : 200_000_000e18 / uint256(1 days)
+            }),
+            "ethenaBurnLimit",
+            18
+        );
+        RateLimitHelpers.setRateLimitData(
+            RateLimitHelpers.makeAssetKey(
+                MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_4626_DEPOSIT(),
+                Ethereum.SUSDE
+            ),
+            Ethereum.ALM_RATE_LIMITS,
+            RateLimitData({
+                maxAmount : 250_000_000e18,
+                slope     : 100_000_000e18 / uint256(1 days)
+            }),
+            "susdeDepositLimit",
+            18
+        );
+        RateLimitHelpers.setRateLimitData(
+            MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_SUSDE_COOLDOWN(),
+            Ethereum.ALM_RATE_LIMITS,
+            RateLimitHelpers.unlimitedRateLimit(),
+            "susdeCooldownLimit",
+            18
+        );
+        RateLimitHelpers.setRateLimitData(
+            RateLimitHelpers.makeAssetKey(
+                MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_4626_WITHDRAW(),
+                Ethereum.SUSDE
+            ),
+            Ethereum.ALM_RATE_LIMITS,
+            RateLimitHelpers.unlimitedRateLimit(),
+            "susdeWithdrawLimit",
+            18
+        );
```

Comment:
This is unused because of the cooldown delay. The forum post has made a mistake `Withdrawing from sUSDe after cooldown` is always unlimited. This should be removed.

Reply:
removed